octobot_trading/exchanges/types/rest_exchange.py

```diff
@@ -264,7 +268,7 @@ async def _create_specific_order(self, order_type, symbol, quantity: decimal.Dec
                                                                       params=params)
         elif order_type == enums.TraderOrderType.STOP_LOSS_LIMIT:
             created_order = await self._create_limit_stop_loss_order(symbol, float_quantity, price=float_price,
-                                                                     side=side, params=params)
+                                                                     side=side, stop_price=stop_price, params=params)
```

Comment:
I don't understand why we need a new param here. Why not using price?

Reply:
stop limit orders have two parameter for price.
price will be the actual limit order price of the limit order.
The limit order gets created by the exchange, when the mark price crosses stop_price.

Reply:
Right :+1: Thanks!